isolate_test.go

```diff
@@ -135,6 +135,68 @@ func TestIsolateGarbageCollection(t *testing.T) {
 	time.Sleep(time.Second)
 }
 
+func TestIsolateThrowException(t *testing.T) {
+	t.Parallel()
+	iso := v8.NewIsolate()
+	defer iso.Dispose()
+
+	strErr, err := v8.NewValue(iso, "some type error")
+
+	if err != nil {
```

Comment:
We can assume that `v8.NewValue` won't return an error here (which is the pattern in the other tests because we can test the error handling in just `value_test.go` instead of everywhere)